Functions/ImageProcessing/xASL_im_IM2Column.m

```diff
@@ -34,8 +34,14 @@
     ApplyShiftDim = true;
 end
 
-if ApplyShiftDim && length(size(ImageIn))>3
-    ImageIn = shiftdim(ImageIn,length(size(ImageIn))-3);
+if length(size(ImageIn))>3
+	if ApplyShiftDim
+		ImageIn = shiftdim(ImageIn,length(size(ImageIn))-3);
+	else
+		% Issue a warning and output all NaNs
+		warning('Cannot accept 4D image when ApplyShiftDim is FALSE');
```

Comment:
Nice, I would then only change this to:

warning('Cannot accept 4D image when ApplyShiftDim is false, check if your data were correctly processed');

Reply:
Done